cmd/syft/internal/options/catalog.go

```diff
@@ -165,6 +168,11 @@ func (cfg Catalog) ToPackagesConfig() pkgcataloging.Config {
 					WithFromBuildSettings(cfg.Golang.MainModuleVersion.FromBuildSettings).
 					WithFromLDFlags(cfg.Golang.MainModuleVersion.FromLDFlags),
 			),
+		DotNet: dotnet.DefaultCatalogerConfig().
+			WithSearchLocalLicenses(cfg.DotNet.SearchLocalLicenses).
+			WithSearchRemoteLicenses(cfg.DotNet.SearchRemoteLicenses).
```

Comment:
We definitely will want to tie these options in to the `enrich` functionality, e.g.:
```
			WithSearchLocalLicenses(*multiLevelOption(false, enrichmentEnabled(cfg.Enrich, task.Dotnet), cfg.DotNet.SearchLocalLicenses)).
			WithSearchRemoteLicenses(*multiLevelOption(false, enrichmentEnabled(cfg.Enrich, task.Dotnet), cfg.DotNet.SearchRemoteLicenses)).
```
... what this does is enable these features with `--enrich all` or a more specific `--enrich dotnet` from the Syft CLI

Reply:
Guess I'll have to look into the ```enrich``` functionality first... 🤷‍♂️